library/couchdb

```diff
@@ -4,15 +4,15 @@
 Maintainers: Joan Touzet <wohali@apache.org> (@wohali), Jan Lehnardt <jan@apache.org> (@janl), Nick Vatamaniuc (@nickva)
 GitRepo: https://github.com/apache/couchdb-docker
 GitFetch: refs/heads/main
-GitCommit: a29d12f0dd4d73f1dcefb07212af4b9ba3185d9e
+GitCommit: 72a0aebfa3248b3df64a70049f0fb1f90c042a49
 
 Tags: latest, 3.3.1, 3.3, 3
```

Comment:
```diff
-Tags: latest, 3.3.1, 3.3, 3
+Tags: latest, 3.3.2, 3.3, 3
```
This needs a tag bump too.

Reply:
Oh my bad. Thanks for noticing, @yosifkit. Fixed it!